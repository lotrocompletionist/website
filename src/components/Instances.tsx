import * as React from "react";

export default function Instances() {
  return <h2>Instances</h2>;
}
