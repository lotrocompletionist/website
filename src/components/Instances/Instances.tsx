import * as React from "react";

import { Instance } from "../Instance/Instance";
import { worldInstances } from "../../data/data";

export default function Instances() {
  return (
    <>
      <h2>Instances</h2>

      {worldInstances.map((worldInstance) => 
        <Instance instance={worldInstance} />)
      }
    </>
  );
}
