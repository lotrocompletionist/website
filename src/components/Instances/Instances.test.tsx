import * as React from "react";
import { create } from "react-test-renderer";
import { default as Instances } from "./Instances";

test("Instances renders correctly", () => {
  const tree = create(<Instances />).toJSON();
  expect(tree).toMatchSnapshot();
});
