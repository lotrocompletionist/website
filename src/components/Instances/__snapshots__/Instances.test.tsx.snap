// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instances renders correctly 1`] = `
Array [
  <h2>
    Instances
  </h2>,
  <div>
    Great Barrow: Thadúr
  </div>,
  <div>
    Great Barrow: Sambrog
  </div>,
  <div>
    Inn of the Forsaken
  </div>,
  <div>
    Garth Agarwen: Arboretum
  </div>,
  <div>
    Garth Agarwen: Barrows
  </div>,
  <div>
    Garth Agarwen: Fortress
  </div>,
  <div>
    Goblin-town Throne Room
  </div>,
  <div>
    Fornost: Wraith of Water
  </div>,
  <div>
    Fornost: Wraith of Earth
  </div>,
  <div>
    Fornost: Wraith of Fire
  </div>,
  <div>
    Fornost: Wraith of Shadow
  </div>,
  <div>
    The Halls of Night
  </div>,
  <div>
    Glinghant
  </div>,
  <div>
    Ost Elendil
  </div>,
  <div>
    Haudh Valandil
  </div>,
  <div>
    Urugarth
  </div>,
  <div>
    Carn Dûm
  </div>,
  <div>
    Barad Gúlaran
  </div>,
  <div>
    The Library at Tham Mírdain
  </div>,
  <div>
    The School at Tham Mírdain
  </div>,
  <div>
    The Dark Delving
  </div>,
  <div>
    The Forgotten Treasury
  </div>,
  <div>
    The Grand Stair
  </div>,
  <div>
    Forges of Khazad-dûm
  </div>,
  <div>
    Fil Gashan
  </div>,
  <div>
    The Sixteenth Hall
  </div>,
  <div>
    Skûmfil
  </div>,
  <div>
    The Mirror-halls of Lumul-nar
  </div>,
  <div>
    The Water Wheels: Nalâ-dûm
  </div>,
  <div>
    Halls of Crafting
  </div>,
  <div>
    Sword-hall of Dol Guldur
  </div>,
  <div>
    Dungeons of Dol Guldur
  </div>,
  <div>
    Warg-pens of Dol Guldur
  </div>,
  <div>
    Sammath Gûl
  </div>,
  <div>
    Lost Temple
  </div>,
  <div>
    Sâri-surma
  </div>,
  <div>
    Northcotton Farms
  </div>,
  <div>
    Stoneheight
  </div>,
  <div>
    Dargnákh Unleashed
  </div>,
  <div>
    Fangorn's Edge
  </div>,
  <div>
    Pits of Isengard
  </div>,
  <div>
    The Foundry
  </div>,
  <div>
    Roots of Fangorn
  </div>,
  <div>
    Seat of the Great Goblin
  </div>,
  <div>
    Iorbar's Peak
  </div>,
  <div>
    Webs of the Scuttledells
  </div>,
  <div>
    The Bells of Dale
  </div>,
  <div>
    Helm's Dike
  </div>,
  <div>
    The Deeping Wall
  </div>,
  <div>
    The Deeping-coomb
  </div>,
  <div>
    The Glittering Caves
  </div>,
  <div>
    The Hornburg
  </div>,
  <div>
    Retaking Pelargir
  </div>,
  <div>
    Defence of Minas Tirith
  </div>,
  <div>
    Hammer of the Underworld
  </div>,
  <div>
    Sunken Labyrinth
  </div>,
  <div>
    The Ruined City
  </div>,
  <div>
    The Dome of Stars
  </div>,
  <div>
    Blood of the Black Serpent
  </div>,
  <div>
    The Quays of the Harlond
  </div>,
  <div>
    The Silent Street
  </div>,
]
`;
